namespaces/api-description-namespace.md

```diff
@@ -177,13 +179,17 @@ Note: At the moment only the HTTP protocol is supported.
         Definition of any input message-body attribute for this transition.
 
     - `contentTypes` (array[String]) - A collection of content types that MAY be used for the transition.
-- `content` (array[HTTP Transaction]) - Array of transaction examples.
+- `content` (array)
+    - (Copy) - Textual information of the transition.
+    - (HTTP Transaction) An instance of transaction example.
+
+        The `content` MAY include multiple `HTTP Transaction` elements.
```

Comment:
Well, the same sentence is getting used everywhere else in the document. Ex: https://github.com/refractproject/refract-spec/pull/41/files#diff-47715162e9c85197ac4ed245268438e6R102